### PR TITLE
walletrpc: Add Taproot witness types to rpc.

### PR DESCRIPTION
**Pull request by ziggie1984**
Fixes https://github.com/lightningnetwork/lnd/issues/8322

Not sure why we did not add the RPC tests to the Unit tests, I guess I am missing something or ?

Otherwise I could not test `TestWitnessTypeMapping` locally with ` make unit  pkg=lnrpc/walletrpc case=TestWitnessTypeMapping`

### CODE REVIEW

File: lnrpc/walletrpc/walletkit.proto [hunk 1]
```diff
@@ -976,6 +976,91 @@ enum WitnessType {
...
+    TAPROOT_HTLC_ACCEPTED_SUCCESS_SECOND_LEVEL = 27;
+
+    /*
+    A witness that allows us to sweep an HTLC on the revoked transaction of the
+    remote party that goes to the second level.
+    */
+    TAPROOT_HTLC_SECOND_LEVEL_REVOKE = 28;
+
+    /*
+        A witness that allows us to sweep an HTLC sent to us by the remote party
```
Comment (guggero) [inline lnrpc/walletrpc/walletkit.proto:1017]: nit: indentation incorrect.

**Review by guggero — APPROVED** [review]
Thanks, LGTM :tada:

File: make/testing_flags.mk [hunk 2]
```diff
@@ -97,19 +97,19 @@ UNIT_TARGETED ?= no
 # If a specific package/test case was requested, run the unit test for the
 # targeted case. Otherwise, default to running all tests.
 ifeq ($(UNIT_TARGETED), yes)
-UNIT := $(GOTEST) -tags="$(DEV_TAGS) $(LOG_TAGS)" $(TEST_FLAGS) $(UNITPKG)
-UNIT_DEBUG := $(GOTEST) -v -tags="$(DEV_TAGS) $(LOG_TAGS)" $(TEST_FLAGS) $(UNITPKG)
-UNIT_RACE := $(GOTEST) -tags="$(DEV_TAGS) $(LOG_TAGS) lowscrypt" $(TEST_FLAGS) -race $(UNITPKG)
+UNIT := $(GOTEST) -tags="$(DEV_TAGS) $(RPC_TAGS) $(LOG_TAGS)" $(TEST_FLAGS) $(UNITPKG)
```
Comment (yyforyongyu) [inline make/testing_flags.mk:100]: hmm 🤦

**Review by yyforyongyu — APPROVED** [review]
LGTM 🌹

**Pull request merged**